scripts/install

```diff
@@ -142,7 +142,12 @@ mkdir -p $final_path
 # For any update do it in all files
 if [ -n "$(uname -m | grep arm)" ]
 then
-    ynh_setup_source $final_path/ "armv7"
+    if [ "$(lsb_release --codename --short)" == "jessie" ]
+    then
+        ynh_setup_source $final_path/ "armv7_jessie"
+    else
+        ynh_setup_source $final_path/ "armv7_stretch"
+    fi
```

Comment:
You could use `ynh_setup_source $final_path/ "armv7_$(lsb_release --codename --short)"` to shorten this code, or use a variable to keep it clear.
But I think it's isn't necessary to keep this if/else structure.

Reply:
Good idea. Thanks